lib/action_controller/serialization.rb

```diff
@@ -56,10 +56,15 @@ def use_adapter?
 
     [:_render_option_json, :_render_with_renderer_json].each do |renderer_method|
       define_method renderer_method do |resource, options|
-        options.fetch(:serialization_context) do
-          options[:serialization_context] = ActiveModelSerializers::SerializationContext.new(request, options)
+        options.freeze
+        serialization_options = options.deep_dup
```

Comment:
wouldn't the serializer be a class at this point?  How would that be a problem.

I suppose an alternative would be something like strong params where we basically specific which params we want, but that would kill the `instance_options` behavior.